src/commands/txCommands/lib.js

```diff
@@ -0,0 +1,266 @@
+const babel = require('babel-core');
```

Comment:
nitpick: since this module isn't a command, can it be moved to `txCommands/util/index.js`? It can then be imported with `require('./txCommands/util');`